openfl-tutorials/interactive_api/Tensorflow_Word_Prediction/envoy/shard_descriptor.py

```diff
@@ -28,13 +46,19 @@ def __init__(self, title: str = '', author: str = '') -> None:
         self.data = self.load_data(self.dataset_dir)  # list of words
         self.X, self.y = self.get_sequences(self.data)
```

Comment:
There are two options:
1. To use `dataset_dir` instead of `self.dataset_dir`
2. To use `self.load_data()` instead of `self.load_data(self.dataset_dir)` and make it general method instead of static

Because we don't need pass class attributes to class methods.
The same for `self.get_sequences`